libs/blocks/merch-card/merch-card.js

```diff
@@ -247,6 +284,35 @@ function createQuantitySelect(el) {
   return quantitySelect;
 }
 
+const getMiniCompareChartFooterRows = (el) => {
+  let footerRows = [];
+  footerRows = Array.from(el.children).slice(1);
+  footerRows.forEach((row) => row.remove());
+  return footerRows;
+};
+
+const decorateFooterRows = (merchCard, footerRows) => {
+  if (footerRows) {
+    const footerRowsSlot = createTag('div', { slot: 'footer-rows' });
+    footerRows.forEach((row) => {
+      const rowIcon = row.firstElementChild.querySelector('picture');
```

Comment:
I think we generated too many DOM elements here.
Couldn't we simply render a css grid here?

Reply:
Done, simplifed DOM